cs3api4lab/utils/file_utils.py

```diff
@@ -46,10 +46,58 @@ def normalize_path(path):
             path = '/'
         return path
 
+    @staticmethod
+    def get_my_shares_path_prefix(config):
```

Comment:
This was discussed by email and it's not correct that the shares folder lives inside the home dir. We also discussed that we cannot use this folder any longer, so I believe this PR needs to be changed.